enterprise/backend/src/metabase_enterprise/semantic_search/index.clj

```diff
@@ -207,15 +223,15 @@
 
 (defn- semantic-search-query
   "Build a semantic search query using vector similarity."
-  [embedding search-context]
+  [index embedding search-context]
   (let [filters (search-filters search-context)
         base-query {:select [[:model_id :model_id]
                              [:model :model]
                              [:content :content]
                              [:verified :verified]
                              [[:raw (str "embedding <=> " (format-embedding embedding))] :distance]
                              [:metadata :metadata]]
-                    :from   [*index-table-name*]
+                    :from   [(keyword (:table-name index))]
```

Comment:
Do we ever use the `:table-name` / `:index-name` as strings? Would it make sense to just create/keep them as keywords?

Reply:
Good point.

I think they are/were but its worth checking again now, I do remember calling `keyword` a bit too much. I'll see if I can change them